vector/src/main/java/im/vector/app/features/call/VectorCallViewModel.kt

```diff
@@ -145,10 +146,17 @@ class VectorCallViewModel @AssistedInject constructor(
         }
 
         override fun onAudioDevicesChange(mgr: WebRtcPeerConnectionManager) {
+            val currentSoundDevice = mgr.audioManager.getCurrentSoundDevice()
+            if (currentSoundDevice == CallAudioManager.SoundDevice.PHONE) {
```

Comment:
Why only do it if on ear peace?

Reply:
We are trying to protect the screen against undesired touches with the ear. Do you think we have another use case?

Reply:
I think there could be some use to switch from earpiece/speaker depending on proximity, but maybe as an option